frontend/components/site/SiteGitHubBranches.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { SITE, GITHUB_BRANCHES } from '../../propTypes';
+import LoadingIndicator from '../loadingIndicator';
+import GitHubRepoLink from '../GitHubRepoLink';
+import BranchViewLink from '../branchViewLink';
+import githubBranchActions from '../../actions/githubBranchActions';
+
+export class SiteGitHubBranches extends React.Component {
+  componentDidMount() {
+    githubBranchActions.fetchBranches(this.props.site);
```

Comment:
I'm not a huge fan of putting this here, but couldn't figure a cleaner place to put it. I wanted to put it in the associated route's `onEnter` prop, but couldn't because it relies on `site`, which may have not been loaded at the time the route is visited.
A better side-effect library (like `redux-sagas`) might help out with this kind of thing, but this is the only weird place right now, so I punted on it.